Automattic/jetpack
# Social | Migrate shares data from initial state to new script data

Currently, we add the shares data to social store which is not needed. We can directly read the data from script data (initial state). That is what this PR does.



## Proposed changes:

* Move the shares related data from social store to the initial state

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to Social admin page
* Confirm that total shares and other shares related data is correct
* Goto the editor on Jetpack, Simple and Atomic sites
* Confirm that editor and Jetpack/Social sidebar loads correctly.
